R/bootstrap.R

```diff
@@ -903,3 +903,32 @@ renv_bootstrap_version_is_dev <- function(version) {
     length(components) != 3
   }
 }
+
+renv_bootstrap_bootstrap <- function(version, library) {
```

Comment:
```suggestion
renv_bootstrap_run <- function(version, library) {
```

Or something like that?